apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -130,32 +130,29 @@ def render(element_html: str, data: pl.QuestionData) -> str:
                         initial_content = empty_diagram
                 else:  # submission
                     initial_content = data["submitted_answers"][drawing_name]
-            else:  # not gradable
-                if fresh and source_available:
-                    initial_content = load_file_content(element, data)
-                else:
-                    raise unreachable
+            elif fresh and source_available:
```

Comment:
we lose a `not gradable` comment, but that doesn't feel like a valuable comment.